server/sonar-db-dao/src/main/java/org/sonar/db/permission/GroupPermissionDao.java

```diff
@@ -65,6 +65,14 @@ public int countGroupsByQuery(DbSession dbSession, PermissionQuery query) {
   }
 
   /**
+   * Select global and project permissions of a given group (Anyone group is NOT supported)
+   * Each row returns a {@link GroupPermissionDto}
+   */
+  public void selectAllPermissionsByGroupId(DbSession dbSession, String organizationUuid, Integer groupId, ResultHandler resultHandler) {
```

Comment:
As discussed offline, I'll keep using ResultHandler in order to stream the result